crates/rmcp/src/model.rs

```diff
@@ -149,7 +149,7 @@ impl ProtocolVersion {
     pub const V_2025_06_18: Self = Self(Cow::Borrowed("2025-06-18"));
     pub const V_2025_03_26: Self = Self(Cow::Borrowed("2025-03-26"));
     pub const V_2024_11_05: Self = Self(Cow::Borrowed("2024-11-05"));
-    pub const LATEST: Self = Self::V_2025_03_26;
+    pub const LATEST: Self = Self::V_2025_06_18;
```

Comment:
IDK, is this accurate to change out?

Reply:
Let's keep it out for now until we support everything in `2025_06_18`

Need to do some automated compliance testing

Sketching out a way to do this with @ochafik here https://github.com/MODELcontextprotocol/MODELcontextprotocol/pull/948